ui/src/main/java/com/epam/eco/kafkamanager/ui/topics/browser/TabularRecords.java

```diff
@@ -147,6 +160,10 @@ public List<Column> listPresentColumns() {
                 collect(Collectors.toList());
     }
 
+    public Set<String> getHeaderFilterLabels() {
+        return headerNames.stream().map(headerName -> HEADER_PREFIX + headerName).collect(Collectors.toSet());
```

Comment:
Maybe, preparing the Set with **HEADER_PREFIX+headerName** values in advance would be more performant instead of doing it each time once again?